week2-section2/exercise2/PlayList.cpp

```diff
@@ -0,0 +1,61 @@
+#include <iostream>
+#include "Song.h"
+#include"PlayList.h"
+
+	FPlaylist::FPlaylist()
+	{
+		PlaylistTitle = "";
+		songCount = 0;
+	}
+	FPlaylist::FPlaylist ( const std::string InPlayListTitle, const int InSongCount )
+	{
+		PlaylistTitle = InPlayListTitle;
+		songCount = InSongCount;
+	}
+	 bool FPlaylist::bAddSong (const FSong InSong ) 
+	{
+		 for (int i = 0; i < songCount; i++)
```

Comment:
With what I commented above, instead of looping through the current songs and using the `SongCount` as the max number of songs, this method could instead use SongCount as an index to add the song if there is still more space for new songs.

This way, you don't have to loop every time. That is why we have the SongCount variable in the first place.